src/styles/HomeStyles.ts

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components/native';
+
+export const TouchableAddButton = styled.TouchableHighlight`
+  padding: 10px;
+  margin: 10px;
+  margin-top: 20px;
+`;
+
+export const TouchableAddView = styled.View`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  padding: 10px;
+  top: 0;
+`;
+
+export const ViewHome = styled.View`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  background-color: white;
+`;
+
+export const TextButton = styled.Text`
```

Comment:
The names are wrong, it should be ButtonText, etc... , also there is no need to especificate what type of components are:
TouchableAddButton should be AddButton
TouchableAddView should be AddContainer, etc